packages/loopring_v3/contracts/impl/Operator.sol

```diff
@@ -37,10 +50,31 @@ contract Operator {
         address payable feeRecipient
         )
         external
+        onlyOperator
     {
         exchange.submitBlocks(
             blocks,
             feeRecipient
         );
     }
+
+    function submitBlocksCompressed(
+        bytes calldata data
+        )
+        external
+        onlyOperator
+    {
+        bytes memory decompressed = LzDecompressor.decompress(data);
+        (bool success, bytes memory returnData) = address(exchange).call(decompressed);
```

Comment:
We should check the bytes corresponding to the function selector in `decompressed` is indeed`submitBlocks`.

Reply:
I don't think that's necessary, the operator contract only has special privileges for calling `submitBlocks` on the exchange contract, nothing else. So I don't think this can be abused in any way.

Reply:
You are right.